### DISCUSSION TITLE
Load test with 3.5 ton AC - soft starter is needed

### DISCUSSION THREAD

**Discussion by zachliu** [jgyates/genmon#1183] [Show and tell]
I'm about to use Genmon to do a load test (`Start Generator and Transfer`) on my Generac 22kw natural gas gen. I assume when we do that, we'd experience a short power outage that last X milliseconds (X = `Transfer switch transition time`)?

Of course I'll maintain a low load during the test (just lights, computers, fridge, etc. No AC.). But I wonder if this is a bad thing for my appliances, depending on the how long the X is. Since it's basically the same as flickering your main circuit breaker on/off.

I did lost an expensive range hood and a desktop computer PSU when accidentally flickering the main circuit breaker in the past...

**Comment by skipfire** [comment]
In my experience it is much faster than a breaker flip and sensitive devices like computers and TVs don't seem to notice.  However, if you have a lot of heavy loads at the time, you will get the bad chug of the generator slamming into all those loads at once.  The generator is also not sync'd with the utility cycle, so if you have loads like AC compressor fans, they may end up slightly out of sync which can make them run rough until they shut down or until the transfer test ends.

All that said, I have mine scheduled for 9:30 on the first of every month and the only thing I take caution with is making sure I don't have the car charging (48A load and the car is touchy about the generator) and the pool heater (electric heat pump) isn't running which is usually only a concern for me on 11/1 and 5/1.

**Comment by jgyates** [comment]
Also, to add to @skipfire's answer, "Start and Transfer" will start the generator, then wait for it to become stable (which takes second or 2) then the transfer switch is activated so there is not "outage". When you Stop a previously started "Start and Transfer" the transfer switch is disengaged, then the generator will continue to run for a cool down period, like it does after an outage, before eventually stoping.

**Comment by zachliu** [comment]
Thank you for the prompt replies! The reason I asked was because I couldn't find the "transition time" spec for my transfer switch. On the other hand, the average "hold-up time" for common home appliances (PCs, TVs, Fridges, LEDs, Washers...) is around 50 milliseconds give or take. So If you didn't see any noticeable interruptions, the transfer switch must achieve ~50 milliseconds transition time, which is close to Generac's [commercial grade transfer switch](https://www.generac.com/globalassets/products/business/energy-solutions/transfer-switches--controllers/industrial-transfer-switches/spec-sheets/671-1000a-open-delayed-transition-industrial-transfer-switch-specsheet.pdf).

**Comment by zachliu** [comment]
I did the load test. Hardly noticed any interruption. But sadly the 3.5 ton AC compressor refused to start :cry:. It kept trying though, for several times. But only ended up "choking" the generator and caused all lights to flickering. In theory, a 22kw generator should be able to handle the operating current (18Amps) but probably not the starting current (112Amps). I'm talking to the generator guy to see what the options are.

**Reply by skipfire** [reply]
I have 17 year old 4 ton ACs and both start fine on my 22kW.  112A on starting current is more 22kW and more than half standard household service, something seems off on that number.

**Reply by zachliu** [reply]
112A is what written on the compressor label I may need to do my own measurements. The generator guy says Generac 22kW units are rated for up to 210 amps LRA :thinking:

**Reply by skipfire** [reply]
That should be amps off the capacitor, not off the generator.  Might want to get the cap checked on the compressor.

**Reply by zachliu** [reply]
Right, gonna measure the inrush current on the compressor first. Depending on the findings, I may need a soft-start kit.

**Reply by zachliu** [reply]
@skipfire I'm curious.. are both your ACs hooked up with this "intelligent load mgmt" system of Generac? 
After all, it was this system that kept killing the compressor. It could be faulty too. 
 
![2024-09-24_16-38](https://github.com/user-attachments/assets/83bf7474-0d53-4217-8f8a-b049997056d7)

**Reply by skipfire** [reply]
All 3 of mine are on there, but that isn't actually smart.  It just has a 5 minute delay on the first wire (add 15 seconds for second, 30 for third, 45 for forth) before it allows the compressor to start by interrupting the cooling wire from the thermostat.  There is no "communication" involved that would make it actually smart.  After that, if the generator chugs and goes below some threshold it redetects it as an outage and retriggers the 5 minute wait.

**Reply by zachliu** [reply]
Ah okay, thanks! So the No.1 priority is still to measure the inrush current and figure out what/why chugs the generator. My HVAC guy is coming today. Hopefully he has a good clamp meter :crossed_fingers:

**Reply by skipfire** [reply]
I notice a small blip in lights and hear the UPSes click when my ACs turn on, but that lasts about half a second and then it's stable until the next on/off event.  The capacitor's purpose is because the load is too much for the house so it buffers a ton of power to be dumped in a very tiny timeframe.  The symptoms you describe sound like that isn't happening on one or both capacitors (many units use double capacitors).  Please do report back what the HVAC guy says, I'm really curious.

**Reply by skipfire** [reply]
Also, if you don't have one, you can get surge suppressors installed at the AC disconnect box, we have dirty utility power where I live and would go through a couple of capacitors a year, after installed the suppressors we went almost 3 years without 1 dying.

**Reply by zachliu** [reply]
I have this surge protector installed 3 years ago. Picture was taken then. Never checked if the protector itself is still alive though.

![surge_protector](https://github.com/user-attachments/assets/b37ad378-6e88-4a67-9a08-e6e58adda636)

**Reply by skipfire** [reply]
My compressors are on the side of my house, so every time I come home at night I can see the 3 green lights.  Makes it really easy to see they are still working.

![image](https://github.com/user-attachments/assets/d5470051-4fb6-495b-87ed-eb752f67dc79)

**Reply by zachliu** [reply]
duh...why didn't i install it on the disconnect box :face_exhaling:

**Reply by Jkh62** [reply]
> I did the load test. Hardly noticed any interruption. But sadly the 3.5 ton AC compressor refused to start 😢. It kept trying though, for several times. But only ended up "choking" the generator and caused all lights to flickering. In theory, a 22kw generator should be able to handle the operating current (18Amps) but probably not the starting current (112Amps). I'm talking to the generator guy to see what the options are.



I installed a hard start kit  on my 3.5 ton Carrier AC unit to make it easier on the generator.

Starts and transfers fine. When I run my monthly 30 minute manual run & transfer I like to have the gen transfer when the AC is not running / loaded. No idea what the contactor surfaces are like in the service disconnect but not a bad idea to take it easy on them for elective monthly test runs.



5-2-1

https://www.cpsproducts.com/hvacr/5-2-1-compressor-saver/



Start load~

https://i.imgur.com/1YJPmSS.jpg

Running load~

https://i.imgur.com/a7SoPaQ.jpg

**Reply by zachliu** [reply]
> > I did the load test. Hardly noticed any interruption. But sadly the 3.5 ton AC compressor refused to start 😢. It kept trying though, for several times. But only ended up "choking" the generator and caused all lights to flickering. In theory, a 22kw generator should be able to handle the operating current (18Amps) but probably not the starting current (112Amps). I'm talking to the generator guy to see what the options are.
> 
> I installed a hard start kit on my 3.5 ton Carrier AC unit to make it easier on the generator.
> 
> Starts and transfers fine. When I run my monthly 30 minute manual run & transfer I like to have the gen transfer when the AC is not running / loaded. No idea what the contactor surfaces are like in the service disconnect but not a bad idea to take it easy on them for elective monthly test runs.
> 
> 5-2-1
> 
> https://www.cpsproducts.com/hvacr/5-2-1-compressor-saver/
> 
> Start load~
> 
> https://i.imgur.com/1YJPmSS.jpg
> 
> Running load~
> 
> https://i.imgur.com/a7SoPaQ.jpg

thanks for the tip! though i'm getting mixed info from the almighty internet :joy: 
it says the industry has been moving away from the "hard starter" in favor of the "soft starter"
some even say "A hard start kit will make it harder to start rather than easier." and "Hard starters Increase the starting current", which would make the situation even worse for the generator

**Comment by Jkh62** [comment]
Higher end HVAC systems use to come with the extra "hard start" cap but many were omitted as a cost save from various accounts of people in the trade. Maybe right , maybe not but hard or soft start does assist on starting load.

My 5-2-1  made a noticeable difference for my AC start up on regular house current , no light flickering or deep kick on sound. Compressor starts a lot "cleaner" or faster. I also replaced the standard start cap with a quality Amrad USA made main capacitor. It has an added lug as a compressor saver that cuts power if the relay sticks.

22KW is a good amount of juice but the LRA off your outside unit plate is the reference number for the amps you need.

**Comment by zachliu** [comment]
Gentlemen, assuming this is a school project, I'm going to use data and facts to settle it.

* AC compressor: Goodman 3.5 ton
    * Key specs from plate:
        * RLA: 17.9 Amp
        * LRA: 112 Amp
    * Measurements using [Klein Tools CL900 Digital Clamp Meter](https://www.amazon.com/dp/B079MV9DRH?ref=ppx_yo2ov_dt_b_fed_asin_title):
        * running current: 11.3 Amp
        * inrush current: 123 Amp
    * HVAC guy
        *  Since the measured inrush is less than 10% larger than the plate LRA, he's not going to do anything. And he doesn't think the capacitor is bad.

* Issue
    * Generac's load mgmt system deems the inrush load (123 A x 240 V = 29.5 kw) too high and cuts off the thermostats 24v signal

* Solution 1: [CSRU2 5-2-1 hard start kit](https://www.amazon.com/dp/B0C2F5GDS9?ref=ppx_yo2ov_dt_b_fed_asin_title) advertises 50% reduction on the inrush current in its [brochure](https://res.cloudinary.com/cps/image/upload/v1683734225/5-2-1_CSRU1_2_3_Brochure_LT52104_Rev_H.pdf)
    * Measured inrush: 106 Amp
    * I didn't test this with the generator

* Solution 2: [Airgo AC soft starter](https://www.amazon.com/dp/B0D4B42QQN?ref=ppx_yo2ov_dt_b_fed_asin_title) doesn't specify inrush current reduction but I hear good things on [reddit](https://www.reddit.com/r/Generator/comments/1eiscg7/has_anyone_here_installed_an_airgo_brand_soft/). Some are too good to be true: 65-70% reduction
    * Measured inrush: 63-70 Amp
    * The generator load mgmt sys said yes :tada: (LED lights are still flickering though, but everything else is fine)

* Notes & References:
    * I did order a good [AmRad capacitor](https://www.amazon.com/dp/B00T3DQZ4U?ref=ppx_yo2ov_dt_b_fed_asin_title) but it didn't arrive today so I'll leave it in reserve for the future
    * [Reddit thread on Airgo](https://www.reddit.com/r/Generator/comments/1eiscg7/has_anyone_here_installed_an_airgo_brand_soft/)
    * [AC Soft Start and Generator Size](https://www.reddit.com/r/Generator/comments/14tnsll/ac_soft_start_and_generator_size/)
    * [Strain on Generac 22kw when AC comes on](https://www.reddit.com/r/Generator/comments/1dnh7iu/strain_on_generac_22kw_when_ac_comes_on/)
    * [22kw Generac Generator wont run 5 ton AC unit](https://www.zillerstore.com/post/22kw-generac-generator-wont-run-5-ton-ac-unit-10276581)

![IMG_1443](https://github.com/user-attachments/assets/be353ad6-174e-41ad-b050-8d7e2dee0d8f)

![IMG_1439](https://github.com/user-attachments/assets/b0938263-8349-467f-9a37-03240e3b5bb8)

**Comment by Jkh62** [comment]
I am not using Generac load management but can attest to the 5-2-1 hard start add on , works fine with my 22KW system.
Hard or soft start they both benefit your generator and take the bite out of the starting load in different ways.

The 5-2-1 is really inexpensive and simple to put on , just mount the relay in the correct direction ( up ).

**Comment by zachliu** [comment]
Ironically, the real culprit is the stupid capacitor :angry:. I swapped it with a new AmRad. It made a huge difference, now the inrush is down to a whopping `33` Amp!

Funny that capacitance-wise the old one is actually closer to its spec 45/5 uF, while AmRad is 46.2/5.1 uF but of course a cap has other characteristics that have huge impact on the inrush. 

I guess it would've worked if I only replaced the capacitor without having to go through all the troubles. But it's too late, I'm settled with Airgo + AmRad now :joy:

**Reply by skipfire** [reply]
I had the same thing happen with my AC on 8/31 (I can see in IoTaWatt power monitoring) and then on 9/1 when my monthly transfer test started I had all sorts of problems and had to cut the breakers to the condenser because it was trying to draw all the amps off the circuit instead of the dead cap.  Turns out the compressor was destroyed by the cap failure, but it was 18 years old and had already had a bunch of problems since we bought the house 6 years ago so we were looking to replace all 3 anyway.  Got the AC replaced and it has a soft-start kit built in and it is really gentle on the generator.  A good capacitor makes the most difference, after that a hard or soft start kit helps even more.

**Reply by zachliu** [reply]
:+1: 

According to chatgpt, brands known for incorporating inverter technology—and thereby offering soft start characteristics—include:

Daikin
Mitsubishi Electric
Fujitsu
LG
Samsung
Panasonic

**Comment by Jkh62** [comment]
Caps are magical devices.
I have rebuilt lots of circuit boards where caps meter in range but failed in circuit.
Same for bridge rectifiers, the diodes sometimes check OK but do not work correctly.

Glad you got your system working.
